### PR TITLE
Added game functionality

### LINKED ISSUE #5: Game Start Functionality
Write the function to start the game, it should contain the following functions

- Timer and moves
- Create icon array and shuffle it
- Create Table

### PR DESCRIPTION
**Pull request by warrenferns**
Closes #5

### CODE REVIEW

File: game.js [hunk 1]
```diff
@@ -39,20 +39,53 @@ window.onload = function() {
...
+        $("#time").html("Time: 0"+min+":0"+sec);
+    else 
+      $("#time").html("Time: 0"+min+":"+sec);
+  }, 1000);
+  rem=r*l/2, noItems=rem;
+  mode = r+"x"+l;
+  //Generating item array and shuffling it
+  var items = [];
+  for (var i=0;i<noItems;i++)
+      items.push(em[i]);
```
Comment (suzandsouza) [inline game.js]: Resolve the uncaught reference error for 'em' on lines 61 and 63

**Review by suzandsouza — COMMENTED** [review]
Resolve the reference error for 'em' and correctly specify the id attribute 'table'

**Review by suzandsouza — COMMENTED** [review]
Game start function and table generation are working all right, gtg

**Comment by warrenferns** [comment]
> Game start function and table generation are working all right, gtg

@suzandsouza plz approve

**Review by suzandsouza — APPROVED** [review]

**Pull request merged**